docs/team/merlinlim.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Merlin Lim's Project Portfolio Page
+---
+
+## Project: Productiv
+
+Productiv is a desktop application for product managers to organise their product-related information. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Delete command for Meeting
+    * Delete command for meeting allows users to delete meeting from list.
+    * This feature allows users to remove meetings which may have been done or cancelled
```

Comment:
Might want to standardise the periods at the end of all your bullet points in this file. Currently some have and some don't. [I personally suggest taking out all periods]